test/functional/tools/inputs_as_json_profile.xml

```diff
@@ -46,7 +46,7 @@ elif test_case == "2":
     assert_equals(as_dict["floattest"], 1.0)
     assert_equals(as_dict["radio_select"], "a_radio")
     assert_equals(as_dict["optional_select"], "a")
-    assert_equals(as_dict["optional_multiple_select"], 'a,b')
+    assert_equals(as_dict["optional_multiple_select"], ['a', 'b'])
```

Comment:
Thanks for the fixes!

Reply:
If this has been broken since 20.05 - should we just change the profile version we check to be 21.05?

Reply:
> If this has been broken since 20.05 - should we just change the profile version we check to be 21.05?

If I'm not wrong:

- we introduced the change in 20.01
- then we decided in 20.05 to "hide" the change in a profile version .. and apparently I messed up that change :( 

My preference would be to backport it down to 20.01 .. then we have the documented behavior in all versions from 20.01. ... would be also good because the OpenMS tool suite depends on the correct export of `None`.

Reply:
I created PRs for 20.05 to 20.01 .. because I was wrong about that the changes started in 20.01 https://github.com/galaxyproject/galaxy/blob/release_20.01/lib/galaxy/tools/parameters/wrapped_json.py